### PR TITLE
Fortran update

### PR DESCRIPTION
**Pull request by theHamsta**
I some how lost the permissions to push on @oponkork master I pushed a wrong commit. The update of the lockfile.json caused the conflict.

### CODE REVIEW

**Comment by theHamsta** [comment]
This is #1437 with rebase conflicts resolved (drop a lockfile commit)

File: queries/fortran/highlights.scm [hunk 1]
```diff
@@ -122,6 +127,8 @@
   ")"
   "["
   "]"
+  "<<<"
+  ">>>"
  ] @punctuation.bracket
```
Comment (stsewd) [inline queries/fortran/highlights.scm:132]: feels weird having these as bracket, maybe special? Not a fortran programmer, so.

**Review by stsewd — APPROVED** [review]

File: queries/fortran/highlights.scm (hunk 1 above)
Comment (theHamsta) [reply queries/fortran/highlights.scm:132]: @oponkork

Comment (theHamsta) [reply queries/fortran/highlights.scm:132]: Ah, it's for CUDA kernels. Then, it's like in C++

**Pull request merged**